File: HA1_5.cpp
//Accept one number from user and print that number of * on screen

#include<iostream>
using namespace std;

void PrintStar(int iNo)
{
    int iCnt = 0;
    for(iCnt =1; iCnt <= iNo; iCnt++)
    {
        cout<<"*";
    }
}

int main()
{
    int iValue = 0;

    cout<<"Enter number"<<"\n";
    cin>>iValue;

    PrintStar(iValue);

    return 0;
}

File: HA1_5.py
#Accept one number from user and print that number of * on screen

def PrintStar(iNo):
    for iCnt in range(0,iNo,1):
        print("*")

def main():
    iValue = 0
    iRet = 0

    print("Enter a number")
    iValue = int(input())

    PrintStar(iValue)

if __name__ == "__main__":
    main()